### PR TITLE
Settings

### PR DESCRIPTION
**Pull request by Reclusior**

### CODE REVIEW

File: Source/PawnPlus.Core/Views/Main.xaml.cs [hunk 1]
```diff
@@ -11,14 +11,15 @@ namespace PawnPlus.Core.Views
...
+        //private XML XMLComponent;
 
         public Main()
         {
             InitializeComponent();
 
-            XMLComponent = new XML();
-            if (System.IO.File.Exists(Info.config) == false)
+           // XMLComponent = new XML();
+            WindowState = Properties.Settings.Default.WindowState;
```
Comment (WopsS) [inline Source/PawnPlus.Core/Views/Main.xaml.cs]: Please use **this.**.

File: Source/PawnPlus.Core/Views/Main.xaml.cs [hunk 2]
```diff
@@ -11,14 +11,15 @@ namespace PawnPlus.Core.Views
...
+            /* (System.IO.File.Exists(Info.config) == false)
```
Comment (WopsS) [inline Source/PawnPlus.Core/Views/Main.xaml.cs]: If you don't need them, please delete them.

File: Source/PawnPlus.Core/Views/Main.xaml.cs [hunk 3]
```diff
@@ -144,15 +143,11 @@ private void Menu_Open_Click_File(object sender, RoutedEventArgs e)
 
         private void PawnPlus_MainFrame_StateChanged(object sender, EventArgs e)
         {
-            if(WindowState == WindowState.Normal)
-            {
-                Info.Settings[Convert.ToInt32(settingslist.lastWindowState)] = "normal";
-            }
-            else if (WindowState == WindowState.Maximized)
+            if(WindowState == WindowState.Normal || WindowState == WindowState.Maximized)
```
Comment (WopsS) [inline Source/PawnPlus.Core/Views/Main.xaml.cs]: You can use operator **?** (inline if-else).

File: Source/PawnPlus.Core/Views/Main.xaml.cs (hunk 2 above)
Comment (Reclusior) [inline Source/PawnPlus.Core/Views/Main.xaml.cs]: I forgot to remove it - yes

File: Source/PawnPlus.Core/Views/Main.xaml.cs (hunk 1 above)
Comment (Reclusior) [inline Source/PawnPlus.Core/Views/Main.xaml.cs]: That wouldn't change anything

Comment (WopsS) [inline Source/PawnPlus.Core/Views/Main.xaml.cs]: I know, but it is more easy to see from where it is with **this.** (I know it is from the current window, but I like it more like this).

File: Source/PawnPlus.Core/Views/Main.xaml.cs (hunk 3 above)
Comment (Reclusior) [inline Source/PawnPlus.Core/Views/Main.xaml.cs]: Sure I can use it, but it won't be necessary.
In a later commit, I will change it.

Comment (WopsS) [inline Source/PawnPlus.Core/Views/Main.xaml.cs]: Thanks!

**Comment by WopsS** [comment]
I will close this because I want to do some refactoring.

**Pull request closed without merging**